common/webui_url_constants.cc

```diff
@@ -10,7 +10,10 @@ const char kRewardsHost[] = "rewards";
 const char kRewardsJS[] = "brave_rewards.js";
 const char kWelcomeHost[] = "welcome";
 const char kWelcomeJS[] = "brave_welcome.js";
+const char kDonateHost[] = "donate";
+const char kDonateJS[] = "brave_donate.js";
 const char kBraveNewTabJS[] = "brave_new_tab.js";
 const char kBraveUIWelcomeURL[] = "chrome://welcome/";
 const char kBraveUIRewardsURL[] = "chrome://rewards/";
 const char kBraveUIAdblockURL[] = "chrome://adblock/";
+const char kBraveUIDonateURL[] = "chrome://donate/";
```

Comment:
is direct navigation to `chrome://donate` expected?

Reply:
cc: @bridiver ^

Reply:
For now, it's only possible in non-OffTheRecord profiles and is non-functional

Reply:
i.e. I blocked it